container.go

```diff
@@ -410,6 +448,8 @@ func (c *Container) start() error {
 		return err
 	}
 
+	c.storeMounts()
```

Comment:
@sboeuf What would we need to refactor ? Once the container is started, we can safely store the bind mount information from outside the agent. Please elaborate.